# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/boltd/hotfix-segfault.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

174       g_autoptr(GOptionContext) context = NULL;

Thread 1 "boltd" received signal SIGSEGV, Segmentation fault.

--- bolt-0.9.8/boltd/bolt-daemon.c.vanilla	2024-07-10 11:46:10.518071869 +0200
+++ bolt-0.9.8/boltd/bolt-daemon.c	2024-07-10 11:47:28.428074247 +0200
@@ -208,7 +208,7 @@
       g_printerr ("Try \"%s --help\" for more information.",
                   g_get_prgname ());
       g_printerr ("\n");
-      g_option_context_free (context);
+      //g_option_context_free (context);
       return EXIT_FAILURE;
     }
 
